models.py

```diff
@@ -205,7 +205,7 @@ def delete(cls, institution, session_name):
 class ServingSession(ndb.Model):
   """Which session is currently serving. Empty if none."""
   session_name = ndb.StringProperty()
-  login_type = ndb.StringProperty()
+  login_type = ndb.StringProperty(choices=['verification', 'preferences', 'schedule', 'preregistration', 'postregistration'])
```

Comment:
Well, this was not originally my code so better ask the author, er arthur. Just kidding. Yes it is, because there is just a single entity (within an institution?) which is updated with a single value.
However, I did notice that institution.html has these same values hardcoded to the checkboxes. We should pass this list to institution.html. But where do we store the list? Now is the time to use a global variable?